edge-middleware/feature-flag-hypertune/.env

```diff
@@ -0,0 +1,4 @@
+HYPERTUNE_FRAMEWORK=nextApp
+HYPERTUNE_PLATFORM=vercel
+HYPERTUNE_OUTPUT_DIRECTORY_PATH=generated
+HYPERTUNE_GET_HYPERTUNE_IMPORT_PATH=../lib/getHypertune
```

Comment:
Is HYPERTUNE_ADMIN_TOKEN missing here?

Reply:
The following environment variables are set via the deploy flow: NEXT_PUBLIC_HYPERTUNE_TOKEN, HYPERTUNE_ADMIN_TOKEN, EDGE_CONFIG, EDGE_CONFIG_HYPERTUNE_ITEM_KEY, FLAGS_SECRET

They are needed in production. The instructions say to pull these to .env.development.local to run the project locally.

The environment variables here in .env are not needed in production. They are used by the CLI tool for codegen only.